cypress/objects/timelog.js

```diff
@@ -6,7 +6,23 @@ cy.url().should('include', '/time-logging');
 pagetitle(){
 return cy.get('.page__title')
 }
-
+calendarv(){
```

Comment:
Is this a typo? :)

Reply:
Name of function?

Reply:
Yeah, 'calendarv' instead of 'calendar' :) But even then it's probably better to add 'get' at the beginning: `getCalendar()`, so that you can immediately understand what the function does when you see it somewhere later.

Reply:
changed! Trying to make right pull request for review.